docpipeline/generate.py

```diff
@@ -22,6 +22,7 @@
 from google.cloud import storage
 from google.oauth2 import service_account
 from google.protobuf import text_format, json_format
+from docpipeline.prepare import prepare_java_toc, add_prettyprint
```

Comment:
Just a thought; doing it like this helps with any other place still relying on `generate.add_prettyprint()` but since `add_prettyprint()` does not originate from `generate` it can seem slightly confusing. Should we change all instances of `generate.add_prettyprint()` to point to the original in `prepare.add_prettyprint()` for the sake of readability?

Reply:
I updated so we always refer to prepare.X in generate.py. Is this what you mean?